bitnami/redis/templates/headless-svc.yaml

```diff
@@ -7,8 +7,12 @@ metadata:
     {{- if .Values.commonLabels }}
     {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
     {{- end }}
+  annotations:
   {{- if .Values.commonAnnotations }}
-  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
```

Comment:
I believe this is a typo.

Reply:
How so? :thinking:

Reply:
This `{{- end }}` should be here. This is the case because `annotations` itself was moved outside of the `if` block statement to support the possibility of no `commonAnnotations`, but present `useExternalDNS` configuration.